.github/workflows/test-suite.yml

```diff
@@ -6,31 +6,31 @@ on:
     - cron: "0 2 * * *" # workflow runs every day at 02:00 AM
   
 jobs:
-  acc-tests:
+
+  mig-tests:
     strategy:
       max-parallel: 1
       fail-fast: false
       matrix:
-        terraform_version: ['${{ vars.TF_VERSION_LATEST }}', '1.0.8']
-    name: acc-tests-${{ matrix.terraform_version }}
+        terraform_version: ['', '1.0.8'] # '' for latest version
+        provider_version: ['', '1.11.1'] # '' for latest version
+    name: mig-tests-${{ matrix.terraform_version || 'latest' }}-${{ matrix.provider_version || 'latest' }}
```

Comment:
nit: in this case when you see the name it won't be intuitive which version is referring to what